### PR TITLE
Welcome Wagon 2019 edits

### PR DESCRIPTION
**Pull request by bethaitman**
@ddbeck - I've had a go at editing this a bit, let me know what you think.

Did we decide to stop doing the Tuesday checkin? I seem to remember it wasn't very well attended.

### CODE REVIEW

**Review by ddbeck — APPROVED** [review]
This looks good to me. And, yeah, I'm fine with dropping the check in.

Want to add the WW events to the main schedule, while you're at it? That would be the only thing I would think to add to this.

**Comment by bethaitman** [comment]
@ddbeck I just seem to remember that nobody came last time. 

Good idea, I'll add that in :)

**Comment by bethaitman** [comment]
@ddbeck done! Can you approve if you're happy?

**Review by ddbeck — APPROVED** [review]
Looks great. Thank you for doing this! 🎉

**Comment by bethaitman** [comment]
Ah, looks like Daniel doesn't have access. @plaindocs, can you give us the okay if this looks alright to you?

**Comment by plaindocs** [comment]
While I'm here, shouldn't the welcome wagon be in the Program not the About menu?

**Comment by bethaitman** [comment]
Could be, I don't have strong opinions about that though

File: docs/conf/prague/2019/welcome-wagon.rst [hunk 1]
```diff
@@ -135,11 +140,11 @@ at the Prague Boats pier no. 5.
 -  If you have a question during a talk, make a note of it and use it as
    a conversation starter with the speaker after the talk.
 -  After a talk, feel free to tweet about it with the hashtag
-   #writethedocs. Try not to "watch" the conference through Twitter and
+   ``#writethedocs``. Try not to "watch" the conference through Twitter and
    other social media, though. You are attending the conference, so live
    in it as much as you can!
 
-**Unconference**
+Unconference:
```
Comment (plaindocs) [inline docs/conf/prague/2019/welcome-wagon.rst:147]: This needs to be a heading (or the other heading doesn't)

**Comment by plaindocs** [comment]
Wanna mention the job fair here instead of the board https://github.com/writethedocs/www/pull/1051/files#diff-0713763707ed9fe5440e2221a00b1cb9R59

**Review by plaindocs — APPROVED** [review]
Job fair nit aside, I'm merging this in.

**Pull request merged**